### PR TITLE
Add Timelab suborg info

### PR DESCRIPTION
**Pull request by matteobachetti**

### CODE REVIEW

File: _projects/2018/timelab/quasiperiodic.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Besides characterizing how these frequencies evolve in time (accelerate, decelerate, for example, when the luminosity increases), 
+it is important to understand their exact physical meaning. A few examples in the literature use different approaches 
+to the problem, and calculate the different spectrum of the light emitted at different phases of these oscillations. 
+This is highly non-trivial: one cannot follow easily the single "heartbeat", but needs to apply a strong de-noising to 
+the signal first, and then calculate the average spectrum during the oscillation. 
+The reward for this scientific approach can be very high, and people have implemented a few different approaches to the problem, e.g.:
+
++ [Ingram et al. 2016](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.461.1967I&link_type=EJOURNAL)
+
++ [Stevens & Uttley 2016](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.460.2796S&link_type=ARTICLE)
```
Comment (dpshelio) [inline _projects/2018/timelab/quasiperiodic.md]: Both links are not open access, I would suggest to also provide the link to ArXiv:
```md
+ Ingram et al. 2016](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.461.1967I&link_type=EJOURNAL) ([open-access preprint](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.461.1967I&link_type=PREPRINT)
+ [Stevens & Uttley 2016](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.460.2796S&link_type=ARTICLE) ([open-access preprint](http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=2016MNRAS.460.2796S&link_type=PREPRINT)
```

File: _projects/2018/timelab/quasiperiodic.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+name: Phase-resolved oscillations
+desc: Implement method to calculate the phase of oscillatory phenomena with non-constant frequency, and calculate phase-resolved spectra.
+requirements:
+ - Familiar with numerical analysis
+difficulty: medium
+issues:
+ - https://github.com/StingraySoftware/stingray/issues/293
+mentors:
+ - matteobachetti
```
Comment (dpshelio) [inline _projects/2018/timelab/quasiperiodic.md]: Will also @abigailStev mentoring this? we prefer 2 mentor per projects if possible (one as a backup)

File: _includes/members/Timelab.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    physics under extreme conditions.
+    <br/>
+ <!-- repo -->
+                <a href="https://github.com/StingraySoftware/" target="_blank">
+              <span class="icon  icon--github">
+                <svg viewBox="0 0 16 16">
+                  <path fill="#333333" d="{{ site.gh_logo }}"/>
+                </svg>
+              </span>
+	      Timelab</a>
```
Comment (dpshelio) [inline _includes/members/Timelab.html]: Maybe you want also to add the slack or any mailing list?

File: _config.yml [hunk 4]
```diff
@@ -56,6 +56,9 @@ ims_repository: https://bitbucket.org/italianmarssociety/eras/src
 coin_url: https://asaip.psu.edu/organizations/iaa/iaa-working-group-of-cosmostatistics/
 coin_repository: https://github.com/COINtoolbox/
 
+timelab_repository: https://github.com/StingraySoftware/
+timelab_url: http://www.timelabtechnologies.com/ideas.html
```
Comment (dpshelio) [inline _config.yml]: Should this be the `home` url, instead of the ideas from last year?

File: _projects/2018/timelab/quasiperiodic.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+In this Project, the student will work with the mentor to implement the full chain of denoising and 
+phase-resolved spectroscopy of quasi-periodic oscillations, in such a way that
+
+Any user with proper data sets can easily get a result
+
+If datasets do not allow a solution (e.g. the modulation is not strong enough) warnings have to be sent.
+
+Both methods can actually be used for pulsars too (it is the trivial case!).
+
+#### Deliverables
```
Comment (dpshelio) [inline _projects/2018/timelab/quasiperiodic.md]: We are also suggesting to break the project into milestones for the three evaluation periods. That helps the students to create better applications.

File: _projects/2018/timelab/quasiperiodic.md (hunk 2 above)
Comment (matteobachetti) [reply _projects/2018/timelab/quasiperiodic.md]: She will actually be the main mentor, I think. I just put my name as a placeholder :)

File: _includes/members/Timelab.html (hunk 3 above)
Comment (matteobachetti) [reply _includes/members/Timelab.html]: Done

File: _projects/2018/timelab/quasiperiodic.md (hunk 5 above)
Comment (matteobachetti) [reply _projects/2018/timelab/quasiperiodic.md]: Done

**Comment by matteobachetti** [comment]
@dpshelio thanks for the review! I addressed your comments, please let me know if there is more to change.
When this project is fully reviewed, we  will add a few more.

File: _projects/2018/timelab/quasiperiodic.md (hunk 1 above)
Comment (abigailStev) [reply _projects/2018/timelab/quasiperiodic.md]: The arxiv is accessible via ADS.

**Pull request merged**